src/lib/constant_mappings.cpp

```diff
@@ -126,4 +126,11 @@ const boost::bimap<AggregateFunction, std::string> aggregate_function_to_string
         {AggregateFunction::CountDistinct, "COUNT DISTINCT"},
     });
 
+const boost::bimap<DataType, std::string> data_type_to_string =
+    make_bimap<DataType, std::string>({{DataType::Int, "int"},
+                                       {DataType::Long, "long"},
+                                       {DataType::Float, "float"},
+                                       {DataType::Double, "double"},
+                                       {DataType::String, "string"}});
```

Comment:
This increases the number of places where we have a list of data types to four. In a perfect world, we could just have a list `string_representation, enum_entry, cpp_type`, e.g.: `"int", Int, (int)`. I think even a macro `ADD_DATA_TYPE()` would help.

Reply:
This is a good idea, should this be done in this pull request though?

Reply:
Done!